Source/Core/VideoCommon/MainBase.cpp

```diff
@@ -125,6 +125,12 @@ void VideoBackendHardware::Video_EndField()
 	if (s_BackendInitialized)
 	{
 		SyncGPU(SYNC_GPU_SWAP);
+		if (s_BackendInitialized && g_ActiveConfig.bUseXFB && SConfig::GetInstance().m_LocalCoreStartupParameter.bCPUThread)
```

Comment:
Please move those conditions within the while loop. At least xfb can be turned off during this loop :/